nengo/builder/learning_rules.py

```diff
@@ -10,7 +10,88 @@
 from nengo.synapses import Lowpass
 
 
-class SimBCM(Operator):
+class IntermittentOperator(Operator):
+    """Base class for operators that support intermittent execution.
+
+    An intermittent operator allows for execution in less than each simulation
+    time step. To implement an intermittent operator derive from this class
+    and ensure that:
+
+    * In the constructor of the derived class you append to the operator lists
+      ``self.sets``, ``self.incs``, ``self.reads``, and ``self.updates`` or, if
+      you decide to overwrite them, include `step` in ``self.reads``.
+    * Set ``self.scaled`` to a list of signals that you want to scale by the
+      period length (``apply_every``) each time step.
```

Comment:
I suppose this won't be applied each time step either, as it's intermittent. How about just "Set ``self.scaled`` to a list of signals that you want to scale by the period length in simulation time steps"